frontend/src/app.config.ts

```diff
@@ -88,7 +88,7 @@ config["eventListenerJobStatus"]["skipped"] = "SKIPPED";
 config["eventListenerJobStatus"]["resubmitted"] = "RESUBMITTED";
 
 config["streamingBytes"] = 1024 * 10; // 10 MB?
-config["rawDataVisualizationThreshold"] = 1024 * 1024 * 10; // 10 MB
+config["rawDataVisualizationThreshold"] = 1024 * 1024 * 1024; // 1 GB
```

Comment:
Can we make the default smaller?

Reply:
Yes, I had planned to keep the previous default as is. This was accidentally committed. I will revert this change.